content/mpi-and-threads-pt1.rst

```diff
@@ -3,31 +3,273 @@ Introducing MPI and threads
 
 .. questions::
 
-   - TODO
+   - What might stop my code scaling?
 
 .. objectives::
 
-   - TODO
+   - Understand the different threading modes available to an MPI rank
+
+
+Why move past pure-MPI programs?
+--------------------------------
+
+MPI is an excellent tool for parallel execution of programs. A key
+strength is that the programmer must explicitly move data to where it
+is needed. That can make code easier to understand, albeit more work
+to write. Since one writes code less often than the author *and
```

Comment:
I had to re-read this sentence 3 times before getting it :smile: Do you think rephrasing as follows keeps the intended meaning?
```suggestion
Since both authors *and* maintainers spend more time **reading** existing code than **writing** new code, that is often desirable.```

Reply:
```suggestion
Since both authors *and* maintainers spend more time **reading**
existing code than **writing** new code, that is often desirable.
```